.github/ISSUE_TEMPLATE/release.md

```diff
@@ -4,32 +4,62 @@ about: Prepare for a release
 labels: maintenance
 ---
 
-- [ ] merge all outstanding PRs
-- [ ] ensure the versions have been bumped (check with `doit`)
-- [ ] ensure the CHANGELOG is up-to-date
-  - [ ] move the new release to the top of the stack
-- [ ] validate on binder
-- [ ] validate on ReadTheDocs
+- [ ] on `dev`
+  - [ ] merge all outstanding PRs
+    - [ ] _blocking #PR here_
+  - [ ] ensure the versions have been bumped
+    - [ ] run `doit preflight:release`
+      - [ ] `pyproject.toml`
+      - [ ] `package.json`
+    - [ ] `README.md` binder badges (`stable` will be broken until tagged)
+  - [ ] ensure `CHANGELOG.md` is up-to-date
+  - [ ] validate on binder
+    - [ ] _URL of binder_
+  - [ ] validate on ReadTheDocs
+    - [ ] _URL of build_
+- [ ] make a PR from `dev` to `main`
 - [ ] wait for a successful build of `main`
-- [ ] download the `dist` archive and unpack somewhere (maybe a fresh `dist`)
+  - [ ] _URL of build_
+- [ ] download the `dist` archive and unpack somewhere
 - [ ] create a new release through the GitHub UI
-  - [ ] paste in the relevant CHANGELOG entries
+  - [ ] pick a release name from real (or imaginary) [forces]
+    - [ ] _name here_
+  - [ ] paste in the relevant `CHANGELOG.md` entries
   - [ ] upload the artifacts
-- [ ] actually upload to npm.com, pypi.org
+- [ ] upload distribution to package repositories
   ```bash
   cd dist
   twine upload *.tar.gz *.whl
   npm login
   npm publish jupyrdf-jupyter-forcegraph-$VERSION.tgz
   npm logout
   ```
+  - [ ] _URL on npmjs.org here_
+  - [ ] _URL on pypi here_
 - [ ] postmortem
-  - [ ] handle `conda-forge` feedstock tasks
+  - [ ] handle `conda-forge` [feedstock] tasks
+    - [ ] _URL on `conda-forge/ipyforcegraph-feedstock` here_
+    - [ ] _URL on `anaconda.org`_
   - [ ] validate on binder via simplest-possible gists
-    - [ ] pip
-    - [ ] conda
-  - [ ] bump to next development version
-  - [ ] bump the `CACHE_EPOCH`
-  - [ ] rebuild `yarn.lock`
-  - [ ] remove `.github/locks` and `doit lock`
-  - [ ] update release procedures with lessons learned
+    - [ ] pip `requirements.txt`
+      - [ ] _URL of pip binder here_
+    - [ ] conda `environment.yml`
+      - [ ] _URL of conda binder here_
+  - [ ] create postmortem PR from `main` targeting `dev`
+    - [ ] _PR# here_
+    - [ ] bump to next development version
+    - [ ] bump the `CACHE_EPOCH`
+    - [ ] rebuild `yarn.lock`
+    - [ ] rebuild `.github/locks`
+      - [ ] run `doit env:lock`
+      - [ ] run `rm -rf .github/locks`
+      - [ ] from a clean command prompt
+        - [ ] 🐧`source envs/lock/bin/activate`
+        - [ ] 🪟 `call envs/lock/Scripts/activate.bat`)
+      - [ ] run `doit lock`
+      - [ ] commit the new locks
```

Comment:
should they re-run all the tests after re-locking?  asking in case a new dependency breaks something, CI should catch that, and probably not necessary to explicitly call for it...